src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenByIntegratedWindowsAuthParameterBuilder.cs

```diff
@@ -46,6 +46,18 @@ public AcquireTokenByIntegratedWindowsAuthParameterBuilder WithUsername(string u
             return this;
         }
 
+        /// <summary>
+        /// Enables MSAL to read the federation metada for a wstrust exchange from a file instead of acquiring it from an endpoint.
+        /// See aka.ms/MsalFederationMetadata
+        /// </summary>
+        /// <param name="fileName">Path to federation metada xml file.</param>
+        /// <returns></returns>
+        public AcquireTokenByIntegratedWindowsAuthParameterBuilder WithFederationMetadata(string fileName)
```

Comment:
Wonder if it would be helpful to have an overload that accepts metadata as and string or an XmlDocument. Although this is probably good for now, since only filepath was asked for.

Reply:
Making this change to accept a string with the metadata instead